tensorflow_serving/sources/storage_path/file_system_storage_path_source.cc

```diff
@@ -76,34 +77,50 @@ std::set<string> GetDeletedServables(
   return deleted_servables;
 }
 
-// Like PollFileSystemForConfig(), but for a single servable.
-Status PollFileSystemForServable(
+void EmitAllVersions(
     const FileSystemStoragePathSourceConfig::ServableToMonitor& servable,
-    std::vector<ServableData<StoragePath>>* versions) {
-  // First, determine whether the base path exists. This check guarantees that
-  // we don't emit an empty aspired-versions list for a non-existent (or
-  // transiently unavailable) base-path. (On some platforms, GetChildren()
-  // returns an empty list instead of erring if the base path isn't found.)
-  if (!Env::Default()->FileExists(servable.base_path())) {
-    return errors::InvalidArgument("Could not find base path ",
-                                   servable.base_path(), " for servable ",
-                                   servable.servable_name());
-  }
+    std::vector<ServableData<StoragePath>>* versions,
+    const std::vector<string>& children) {
```

Comment:
Please move 'children' before 'versions' in the argument order. (Sorry, this is a Google style convention.)